packages/v4/patternfly-docs/content/design-guidelines/components/banner/banner.md

```diff
@@ -7,33 +7,33 @@ section: components
 
 PatternFly offers 5 different banner types detailed below.
  
-<img src="./img/banner-types.png" alt="5 different types of banners" width="744"/>
- 
-|Type     |Background color    |Text color  | Usage
-|---------|---------------|----------------| ------------------ |
-|Default  |Grey <br />(--pf-global--BackgroundColor--dark-400) | Black |Use for generic messages with no associated severity
-|Info     |Blue <br />(--pf-global--info-color--100)|Black| Use for general information messages |
-|Danger  |Red <br />(--pf-global--danger-color--100) |White| Use for high severity messages
-|Success|Green <br />(--pf-global--success-color--100) | White | Use for positive, success messages|
-|Warning |Orange <br />(--pf-global--warning-color--100) | Black | Use for warning or medium severity messages |
-<br />
-PatternFly suggests that users adopt one of these 5 colors, as they’ve been tested against their text colors for accessibility. However, if colors outside of these suggestions are used, we encourage using discretion when selecting the inner text color.
+<img src="./img/banner-types.png" alt="5 different types of banners"/>
+ 
+|Type |Background color |Text color | Usage |
+|-|-|-|-|
+| Default | `--pf-global--BackgroundColor--dark-400` | White | Generic messages with no associated severity |
+| Blue | `--pf-global--info-color--100` | Black | General information messages |
+| Red | `--pf-global--danger-color--100` | White | High severity messages
+| Green | `--pf-global--success-color--100` | White | Positive, success messages |
+| Gold | `--pf-global--warning-color--100` | Black | Warning or medium severity messages |
```

Comment:
Maybe not within the table itself, but perhaps in some context surrounding it, would it be worth mentioning something along the lines of "When used to convey status..." or maybe updating the header to "Usage (status)"/"Usage as status"?

Basically to convey that there are 5 color types and each color has a respective status, but color does not inherently === status